layers/stp.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"github.com/google/gopacket"
+)
+
+// Spanning Tree Protocol is on top of LLC to transport BPDU (bridge protocol data unit) message.
```

Comment:
Godoc comments should start with the name of the thing they're commenting on.  For here:

```go
// STP decodes spanning tree protocol packets to ...
```

Reply:
done